index.html

```diff
@@ -0,0 +1,128 @@
+<html>
+  <head>
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </header>
+      <figure id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="People working"
+        />
+        <p id="img-caption">
```

Comment:
Great Job! Adding an alt text is a good HTML practice. But please it would be better if you try to use class instead id because a class can be used multiple times on different elements throughout a website.